py3status/modules/do_not_disturb.py

```diff
@@ -112,14 +109,17 @@ def toggle(self, state):
         self.iface.SetProperty("xfce4-notifyd", "/do-not-disturb", state)
 
 
+class Meta:
+    deprecated = {"remove": [{"param": "pause", "msg": "obsolete parameter"}]}
```

Comment:
Make sure to update `Dunst Miscellaneous:` section in the docstring in the top of the file